go.mod

```diff
@@ -44,5 +44,6 @@ require (
 	google.golang.org/api v0.43.0
 	google.golang.org/genproto v0.0.0-20210331142528-b7513248f0ba
 	google.golang.org/grpc v1.36.1
+	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.1.0
```

Comment:
It won't let me leave comments on the rest of this file, but I'm interested in why the now-removed stuff (`github.com/golang/protobuf `) is still in the mod file?

Reply:
It's in the generated protobufs? Which is a bit of a mystery, but as an experiment, I tried changing them by hand, and that still left the `github.com/golang/protobuf` dependency, it only got an `// indirect`, so I figured I'd keep this bit for a second pass, as it didn't make it fully go away even if I did it...